pinc/faq.inc

```diff
@@ -35,7 +35,7 @@ $external_faq_overrides = [
         "fr" => "https://www.pgdp.net/wiki/DP_Official_Documentation:CP_and_PM/French/FAQ_fourniture_de_contenu",
         "pt" => "https://www.pgdp.net/wiki/DP_Official_Documentation:CP_and_PM/Portuguese/Fornecimento_de_Conte%C3%BAdos",
     ],
-    "faq_central.php" => [
+    "faq_central" => [
```

Comment:
This needs to remain `faq_central.php` because that key is used elsewhere (see `pinc/theme.inc`). Also so it's parallel to the other uses.

Reply:
Good catch, reverted the key change.